build_manylinux_wheels.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3.5
+"""Script for building 'manylinux' wheels for libsass.
+
+Run me after putting the source distribution on pypi.
+
+See: https://www.python.org/dev/peps/pep-0513/
+"""
+import contextlib
+import os
+import pipes
+import shutil
+import subprocess
+import tempfile
+
+from twine.commands import upload
+
+
+@contextlib.contextmanager
+def tempdir():
```

Comment:
Since you're running on Python 3; there's already a [`TemporaryDirectory` context manager](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory).